### DISCUSSION TITLE
Ability to load textual inversion concepts

### DISCUSSION THREAD

**Discussion by dmt-hub** [hlky/stable-diffusion-webui#824] [Feature suggestions]
There's a library of textual inversion concepts made by community members here: https://huggingface.co/sd-concepts-library

Would be great if we could load these via the webui.

Any possibility?

Thank you.

**Comment by aitrepreneur** [comment]
Would also like to know if there is a way to import these concepts into the webui, I tried importing the .bin or even renaming it into .pt to no avail.

**Reply by underlines** [reply]
Oh hi @Ohmnito , like your YT channel. Hope to see some mentioning of Loab, the Woman haunting inversive prompts :D I didn't find her in SD yet.

**Reply by aitrepreneur** [reply]
@underlines Haha thanks, actually that's not a bad idea I'll put it on the list ;)

**Reply by underlines** [reply]
Cool. :) And sorry I meant negatively weighted prompts, not inversive prompts.  Got carried away finding you here :D

**Reply by aitrepreneur** [reply]
Yeah no worries :)
